tests/specs/opcodes/verification.k

```diff
@@ -1,7 +1,9 @@
 requires "edsl.md"
+requires "../int-simplification.k"
```

Comment:
```suggestion
requires "lemmas/int-simplification.k"
```

Reply:
the suggestions you made along these lines doesn't appear to actually work, I get a build error that says it can't find the file `lemmas/int-simplification.k`. I decided to use the relative path because that's what was used for `lemmas.k` and `infinite-gas.k`